pkgs/applications/window-managers/compton/git.nix

```diff
@@ -0,0 +1,53 @@
+{ stdenv, fetchgit, asciidoc, dbus, docbook_xml_dtd_45,
+  docbook_xml_xslt, libconfig, libdrm, libxml2, libxslt, mesa, pcre,
+  pkgconfig, libXcomposite, libXdamage, libXext, libXfixes, libXinerama,
+  libXrandr, libXrender }:
+
+stdenv.mkDerivation {
+  name = "compton-git";
```

Comment:
please add an appropriate date to the name, instead of a version, e.g. "compton-git-2015-04-23"